app/src/main/java/com/example/myapplication/home/newsNetwork/NewsClinet.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.myapplication.home.newsNetwork
+
+import com.example.myapplication.model.NewsResponse
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class NewsClinet: RemoteDataSource {
+    companion object {
+        private var myInstance: NewsClinet? = null
+        fun getInstance(): NewsClinet? {
+            if (myInstance ==null)
+                myInstance = NewsClinet()
+            return myInstance
+        }
+    }
+    object MyRetrofit {
```

Comment:
move this to Retrofit Client we should add all Retrofit objects in the same class